FirebaseAppDistributionInternal/Sources/FeedbackViewController.swift

```diff
@@ -14,9 +14,33 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class FeedbackViewController: UIViewController {
+  var viewDidDisappearCallback: () -> Void = {}
+  var image: UIImage?
+  var additionalInfo: String?
+
+  @IBOutlet var screenshotUIImageView: UIImageView!
+  @IBOutlet var additionalInfoLabel: UILabel!
+  @IBOutlet var feedbackTextView: UITextView!
+
+  @IBOutlet var navigationBar: UINavigationBar!
+
   override func viewDidLoad() {
     super.viewDidLoad()
     // Do any additional setup after loading the view.
   }
+
+  override func viewWillAppear(_ animated: Bool) {
+    screenshotUIImageView.image = image
```

Comment:
This needs to nil the property, otherwise it could be incorrectly reused the next time this VC shows up.